src/main/java/com/javadiscord/javabot/commands/user_commands/Google.java

```diff
@@ -0,0 +1,27 @@
+package com.javadiscord.javabot.commands.user_commands;
+
+import com.javadiscord.javabot.commands.SlashCommandHandler;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyAction;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+public class Google implements SlashCommandHandler {
+    @Override
+    public ReplyAction handle(SlashCommandEvent event) {
+        String query;
+        String url;
+        try {
+            query = URLEncoder.encode(Objects.requireNonNull(event.getOption("query")).getAsString(), StandardCharsets.UTF_8.toString());
+        } catch (UnsupportedEncodingException e) {
+            e.printStackTrace();
+            return event.reply(e.getMessage());
+        }
+        query = query.trim().replace(" ", "+");
+        url = "https://www.google.com/search?q=".concat(query);
```

Comment:
Instead of just returning the raw search url, wouldn't it be better to list the top 3/5 search results instead? 
Guess google provides some sort of API for this.

Reply:
https://serpapi.com/ ?

Reply:
As Google doesn't provide an API for stuff like this we have decided to rename the command to `/search` and use the [Bing Search API](https://www.microsoft.com/en-us/bing/apis/bing-web-search-api)